docs/src/pages/docs/api/getSimilarity.en.mdx

```diff
@@ -0,0 +1,24 @@
+import { GetSimilarityDemo } from '@/components/demo/get-similarity-demo';
+
+# getSimilarity
+
+Calculates the similarity between two Korean strings and returns a value between 0 and 1. This function uses the disassembleHangul method from the es-hangul library to break down the Korean strings into their component characters (jamo) and then calculates the minimum edit distance between the strings using the Levenshtein distance algorithm. The similarity is then calculated based on this distance.
+
+```tsx
+function getSimilarity()
+  left: string,
+  right: string
+): number;
+```
```

Comment:
```suggestion
```tsx
function getSimilarity(
  // First Korean string to compare
  left: string,
  // Second Korean string to compare
  right: string
): number;
```
```